### DISCUSSION TITLE
I need a component like this

### DISCUSSION THREAD

**Discussion by idear2021** [joomla/joomla-cms#35167] [Q&A] [answered]
Joomla will generate a huge installation package after each backup. The remote host cannot afford the installation package capacity. I hope a component can transfer the JS, CSS and other large files of the website to other places on the network to reduce the load of installation package capacity and operation. Is there any way?

**Comment by ssnobben** [comment, marked as answer]
Hi idear2021 glad to see your first post in Joomla world. 

To correct you Joomla it self doesnt generate a huge installation package after each backup thats a false statement. 

If you after a couple of years have a large Joomla site with millions of articles and user you are right but then you also have understand how Joomla works. Its not problem with any installation files for any good hosting company or organisation or person that want to install Joomla on ordinary conditions.

If you want to use Joomla automatic backup system and also send it to prefered storage outside your own system you can download for example a "component" like Akeeba Backup Pro and set it up easy as you like. That solve all your thoughts so dont worry to use Joomla 4 (incredible) now!

https://www.akeeba.com/products/akeeba-backup.html<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35166">issues.joomla.org/tracker/joomla-cms/35166</a>.</sub>
